setup.py

```diff
@@ -20,36 +31,133 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from setuptools import setup
-from os import path
+# Note: To use the 'upload' functionality of this file, you must:
+#   $ pip install twine
+
+import io
+import os
+import sys
+from shutil import rmtree
+
+from setuptools import find_packages, setup, Command
+
+# Package meta-data.
+NAME = 'octopose'
+DESCRIPTION = 'Command line tools for managing local and remote octopus deploys using manifests.'
+URL = 'https://github.com/HuddleEng/octopose'
+EMAIL = 'george.ayris@huddle.com'
```

Comment:
would update this

Reply:
do you have `oss@huddle.com`?

Reply:
I don't think we have a specific email for oss but you're right, we should probably update it to an active huddle email at least.

Reply:
yeah I find out what it is